src/styles.scss

```diff
@@ -44,7 +44,7 @@ $vue-content-placeholders-spacing: 10px !default;
     width: 100vw;
     max-width: 1000px;
     height: 100%;
-    background: linear-gradient(to right, transparent 0%, darken($vue-content-placeholders-secondary-color, 5%) 15%, transparent 30%);
+    background: linear-gradient(to right, rgba(darken($vue-content-placeholders-secondary-color, 5%), 0) 0%, darken($vue-content-placeholders-secondary-color, 5%) 15%, rgba(darken($vue-content-placeholders-secondary-color, 5%), 0) 30%);
```

Comment:
I think we could also specify a color with a transparency value of zero, and it would still work in Safari:
`rgba(255, 255, 255, 0)`

But your code would also work as well (a very very faint color at 5% opacity).